test/common/src/main/kotlin/org/codice/compliance/utils/TestCommon.kt

```diff
@@ -29,13 +29,15 @@ import org.opensaml.saml.saml2.core.impl.IssuerBuilder
 import org.opensaml.saml.saml2.core.impl.NameIDPolicyBuilder
 import java.io.File
 import java.net.URLClassLoader
-import java.util.*
+import java.util.ServiceLoader
 import kotlin.reflect.KClass
 
 class TestCommon {
     companion object {
         const val XSI = "http://www.w3.org/2001/XMLSchema-instance"
         const val ELEMENT = "http://www.w3.org/2001/04/xmlenc#Element"
+        const val SAML_NAMESPACE = "urn:oasis:names:tc:SAML:2.0:assertion"
```

Comment:
✏️ Since this has to do with an "assertion", should this be named more specifically `ASSERTION_NAMESPACE`?

Reply:
We could. I called it SAML because in core it defines it as 
`saml:` | `urn:oasis:names:tc:SAML:2.0:assertion`